WooCommerce/Classes/ViewRelated/ListSelector/PaginatedListSelectorViewController.swift

```diff
@@ -36,6 +36,12 @@ protocol PaginatedListSelectorDataSource {
     func sync(pageNumber: Int, pageSize: Int, onCompletion: ((Result<Bool, Error>) -> Void)?)
 }
 
+protocol DraggablePaginatedListSelectorDataSource {
```

Comment:
nit: it might be helpful if the protocol has a comment that reminds us that the table view cells are draggable when implementing this protocol and with `PaginatedListSelectorViewController`?